lib/opal/builder.rb

```diff
@@ -223,8 +220,77 @@ def compiled_source(with_source_map: true)
       compiled_source
     end
 
+    # Builds and watches all paths for changes and then rebuilds, does not return
+    def watch
```

Comment:
Can this, and the rest of the methods related to watching, be moved to `Builder::Watcher` which would be included here?